samples/common.js

```diff
@@ -0,0 +1,23 @@
+const Location = {
```

Comment:
We already have all these definitions in the top-level Functions-request-config.js, let's not repeat them again but create a shared, top-level common file.